make dashboard

kongeebowl opened the pull request.

kongeebowl committed:
eee

kongeebowl committed:
donr

kongeebowl committed:
Update main.css

kennething commented on pages/index.vue:
```diff
@@ -1,10 +1,147 @@
+
+const createArticle = async () => {
+  try {
+    const article: Article = await $fetch(`cms/`, {
```
:shipit:

kennething commented on components/Rows/ArticleRow.vue:
```diff
@@ -0,0 +1,63 @@
+      <div class="flex items-center">
+        <div class="ml-4">
+          <div class="text-sm font-medium text-gray-900">
+            {{ article.title.substring(0, 25) }}
```
use line-clamp instead

kongeebowl committed:
aewa

kongeebowl committed:
Refactor tab click handlers to use function declarations

Converted arrow functions to function declarations for onTabClick in Tab.vue and tabClicked in index.vue to improve code consistency and readability.

kennething reviewed (dismissed):
lgtm probably

kongeebowl committed:
a

kongeebowl committed:
Refactor dashboard UI and improve table components

Updated dashboard table and row components for improved layout, styling, and code clarity. Replaced DashboardRowsArticleRow with DashboardRowsArticle, adjusted table structure, and enhanced empty state handling. Refined CSS transitions, fixed minor issues in TabPanel and User row, and improved sign-in endpoint usage.

kongeebowl committed:
Merge branch 'nuxt-3' into 48-make-navbar

kongeebowl committed:
eslint is boooring

kongeebowl committed:
i did it

kongeebowl committed:
uawjnjiodfujhfjio

kongeebowl committed:
Update Header.vue

kongeebowl committed:
Update customizeIcon.ts

kennething reviewed (approved):
<img width="322" height="157" alt="image" src="https://github.com/user-attachments/assets/0760894d-052e-4a8d-8e9d-4ef2bbe33fc7" />
